addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -208,8 +224,7 @@ export class Powerbox {
             }
             const term = this._lastText;
 
-            this._currentFilteredCommands =
-                term === '' ? this._currentOpenOptions.commands : await onValueChangeFunction(term);
+            this._currentFilteredCommands = await onValueChangeFunction(term);
```

Comment:
Why the change ? Is it a bugfix ?

Reply:
we need to pass in the filter every time to ensure the commands who should not appear in inline context are filtered out.